types/types.go

```diff
@@ -314,6 +315,11 @@ type Digest struct {
 	UsersFile string
 }
 
+// Forward authentication
+type Forward struct {
+	Address string
```

Comment:
I think it would be great to add the possibility to connect in TLS (with an self-signed cert).

Reply:
Yes I agree that would be useful. Unfortunately I'm not too familiar with Go yet and not really sure how to go about doing that - would anyone else be interested in implementing that?

Reply:
No problem @drampelt.
We can do the modifications for your :wink: